examples/decoding/plot_mixed_gambles_space_net.py

```diff
@@ -0,0 +1,45 @@
+"""
+SpaceNet on Jimura et al "mixed gambles" dataset.
+==================================================
+
+The segmenting power of SpaceNet is quite visible here.
```

Comment:
Please explain here that you are using the regressor object given that the task is to predict a continuous variable, the gain of the gamble.

Also, maybe we should leave out some data and show a prediction power (if it somewhat works).